### PR TITLE
Fix fs.constants

### PR DESCRIPTION
**Pull request by jetexe**
Make stub for `fs.constants` for web
Fix: #60

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## [Unreleased](https://github.com/json-schema-tools/dereferencer/compare/master...1.2.1)
```
Comment (BelfordZ) [inline CHANGELOG.md:1]: Changelog gets automatically updated =P

**Comment by BelfordZ** [comment]
Commits must follow the conventional commit format: https://github.com/json-schema-tools/dereferencer/blob/master/CONVENTIONAL_COMMITS.md

**Comment by BelfordZ** [comment]
please use 1 commit, with the following: 
```
fix: fs.constants updated for web

resolves #61
```

**Review by BelfordZ — CHANGES_REQUESTED** [review]
also please rebase, as I've updated some deps.

File: CHANGELOG.md (hunk 1 above)
Comment (jetexe) [reply CHANGELOG.md]: fixed

**Pull request merged**